src/TwigComponent/doc/index.rst

```diff
@@ -840,6 +840,117 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
+    
+Component HTML Syntax
+-----------------
+    
+.. versionadded:: 2.9
+
+    This syntax was been introduced in 2.8 and is still experimental: it may change in the future.
+    
+Twig Component came with a new syntax to ease the readability of your template:
+
+.. code-block:: html+twig
+
+    {# templates/some_page.html.twig #}
+    <twig:Alert warning :user="user.username" message="I am an alert!">
+       <div class="alert-content">
+            // ... your component content
+       </div>
+       <twig:block name="footer">
+            // ... footer content
+       </twig:block>
+    </twig:Alert>
+    
+You can now use a custom tag prefix starting by <twig: and your component name
+
+.. code-block:: html+twig
+
+    <twig:Alert></:Alert>
+    // and also self closing tag
+    <twig:Alert/>
+    
+You can pass an attribute to your component like any HTML attribute
+
+.. code-block:: html+twig
+
+    // Warning will be set to true
+    <twig:Alert warning></:Alert>
+    
+    <twig:Alert message="hello" :user="user"/>
+    
+    //camel camelCase works too for your properties names
+    <twig:Alert withActions message="hello" :user="user" />
+    
+And then as usual you can access your properties in your component template
```

Comment:
I *think* we don't need this sentence or the code below, because we'll now show them the PHP component so they can see how the attributes are being set into the properties, like any other component.